store/storetest/group_store.go

```diff
@@ -581,15 +582,24 @@ func testUpsertMember(t *testing.T, ss store.Store) {
 	require.Zero(t, d2.DeleteAt)
 
 	// Duplicate composite key (GroupId, UserId)
-	_, err = ss.Group().UpsertMember(group.Id, user.Id)
+	// MySQL's implementation of RowsAffected returns 0 if the
```

Comment:
Contrary to most unit-testing guidelines, I /feel/ like a simpler solution is just to `time.Sleep(1 * time.Millisecond)` with a comment ala `// Ensure UpdateAt > CreateAt`.